crates/bevy_app/src/app.rs

```diff
@@ -488,84 +363,59 @@ impl App {
     /// # fn system_b() {}
     /// # fn system_c() {}
     /// #
-    /// app.add_system_set(
-    ///     SystemSet::new()
-    ///         .with_system(system_a)
-    ///         .with_system(system_b)
-    ///         .with_system(system_c),
-    /// );
+    /// app.add_systems((system_a, system_b, system_c));
     /// ```
-    pub fn add_system_set(&mut self, system_set: SystemSet) -> &mut Self {
-        self.add_system_set_to_stage(CoreStage::Update, system_set)
+    pub fn add_systems<P>(&mut self, systems: impl IntoSystemConfigs<P>) -> &mut Self {
+        let mut schedules = self.world.resource_mut::<Schedules>();
+
+        if let Some(default_schedule) = schedules.get_mut(&*self.default_schedule_label) {
+            default_schedule.add_systems(systems);
+        } else {
+            let schedule_label = &self.default_schedule_label;
+            panic!("Default schedule {schedule_label:?} does not exist.")
+        }
+
+        self
     }
 
-    /// Adds a system to the [`Stage`] identified by `stage_label`.
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// # use bevy_app::prelude::*;
-    /// # use bevy_ecs::prelude::*;
-    /// #
-    /// # let mut app = App::new();
-    /// # fn my_system() {}
-    /// #
-    /// app.add_system_to_stage(CoreStage::PostUpdate, my_system);
-    /// ```
-    pub fn add_system_to_stage<Params>(
+    /// Adds a system to the provided [`Schedule`].
+    pub fn add_system_to_schedule<P>(
```

Comment:
should this be `add_system_to_schedule(label, system)` instead of `add_system_to_schedule(system, label)`? that would match the add_system_to_stage api.

Reply:
I think the existing API is backwards, and took this opportunity to deliberately flip it.

The order of the arguments should match the name of the API: first a system, then a schedule.